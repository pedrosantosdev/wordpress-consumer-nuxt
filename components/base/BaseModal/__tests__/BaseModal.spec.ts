import { render } from '@testing-library/vue'
import BaseModalVue from '@/components/base/BaseModal/BaseModal.vue'

describe('BaseModalVue', () => {
	it('should render properly', () => {
		const wrapper = render(BaseModalVue, {
			props: {
				showModal: true,
			},
		})
		expect(wrapper.html()).toMatchSnapshot()
	})
})
