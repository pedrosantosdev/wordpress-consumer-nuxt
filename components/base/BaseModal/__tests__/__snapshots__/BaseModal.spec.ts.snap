// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BaseModalVue > should render properly 1`] = `
"<!--teleport start-->
<!--teleport end-->"
`;
